dataprofiler/tests/profilers/test_float_column_profile.py

```diff
@@ -580,12 +583,12 @@ def test_profile(self):
                 'bin_edges': np.array([2.5, 5.0, 7.5, 10.0, 12.5]),
             },
             quantiles={
-                0: 3.75,
-                1: 5.0,
-                2: 10.0
+                0: 4.375,  # 75% between 2.5 and 5 (histogram based)
+                1: 6.25,  # 50% between 5.0 and 7.5 (histogram based)
```

Comment:
so, the median is still 6.25 with the new change? I'd expect some value closer to 5

Reply:
correct, this is because the histogram has only 4 bins. The best it can do is to interpolate between the values of the bins. The real value is 5, but the bin spacing is too large to accurately interpolate / predict.